torchvision/transforms/v2/functional/_geometry.py

```diff
@@ -190,17 +190,14 @@ def resize_image_tensor(
         if interpolation == InterpolationMode.NEAREST or interpolation == InterpolationMode.NEAREST_EXACT:
             # uint8 dtype can be included for cpu and cuda input if nearest mode
             acceptable_dtypes.append(torch.uint8)
-        elif interpolation == InterpolationMode.BILINEAR and image.device.type == "cpu":
+        elif (
+            interpolation == InterpolationMode.BILINEAR
+            and image.is_cpu
```

Comment:
I can attest that this actually works, but

1. I've never seen used it anywhere
2. it is [undocumented](https://pytorch.org/docs/main/search.html?q=is_cpu). This looks like a bug though, since `is_cuda` and `is_meta` are there.

Thus, I would prefer to leave it as is, but no strong opinion. How did you learn about it?

Reply:
```
  torchvision/transforms/v2/functional/_geometry.py:195: error: "Tensor" has no
  attribute "is_cpu"  [attr-defined]
                  and image.is_cpu
                      ^~~~~~~~~~~~
  Found 1 error in 1 file (checked 236 source files)
  Error: Process completed with exit code 1.
```


bwhahahaha.
Anyway.


> How did you learn about it?

I was reminded that `is_cuda` exists so I sort of guessed `is_cpu` should exist as well. It's used in the torch core code-base, but sparsely. I'll revert anyway to avoid fighting `mypy`